R/read_file.R

```diff
@@ -10,11 +10,12 @@
 #'  * `.parquet` uses [arrow::read_parquet()].
 #'
 #' @param path The file path to be read
+#' @inheritParams arrow::read_parquet
 #' @param ... Addition arguments passed to the relevant function.
 #'
 #' @return the data a [tibble][tibble::tibble-package]
 #' @export
-read_file <- function(path, ...) {
+read_file <- function(path, col_select = NULL, as_data_frame = TRUE, ...) {
```

Comment:
I tested this with the code: `ep1718 <- read_file(slf_path, col_select = "recid")` but this still read in all of the variables. Have i done this wrong? Other than that i think the changes on this PR look good

Reply:
Nope, not wrong! I should have fixed it in this commit: https://github.com/Public-Health-Scotland/source-linkage-files/pull/693/commits/93ab9ed71028225bb3ad58ec908cc1299bac6128

Reply:
Working as expected!